deps.list

```diff
@@ -3,8 +3,6 @@ GO_VERSION=1.19.3
 GOOGLE_API_VERSION=9de27c9c549173eac194ec80153963234a05a8b5
```

Comment:
Can we drop this merge commit?

Reply:
Sorry about that! I'm a relative novice at git, so I'm accustomed to messing around on branches and using squash merges to make it look nice afterwards. I'd love to learn better git hygiene, we can definitely drop this if you do it (or show me how)

Reply:
Rebased on top of `origin/main` and squashed fixups into feature commit (e42e045)